### ISSUE TITLE
What is the added value of totalItems?

### ISSUE THREAD

**Issue by kbrueckmann** [distributed-text-services/specifications#248]
A Collection currently has the properties _totalChildren_, _totalParents_ and _totalItems_. If I understand the documentation correctly, _totalItems_ either has the value of _totalChildren_ (if we do nav=children, which is the default behaviour) or it has the value of _totalParents_ (if we do nav=parents). So I'm wondering if the usage of _totalParents_ and _totalChildren_ would allow us to drop the required _totalItems_, as it doesn't seem to add any new information. Please correct me if I misunderstood anything. 

Might be connected to #141 (historical development of the 3 properties :))

**Comment by PonteIneptique** [comment]
Thanks @kbrueckmann. We just discussed this

**Decision of May 24th:** Drop `totalItems`

**Rationale:** `totalItems` is a remain of a time where DTS wanted Hydra compliance. We added totalParents and totalChildren for clarity, as seen in #141. Since `1-alpha` dropped Hydra compliance and just remains "inspired by Hydra", it does make sense to remove `totalItems`.

Thanks for spotting this and asking the question.
